scripts/README.md

```diff
@@ -38,6 +38,7 @@ The following steps must be proposed as a pull request.
 - [ ] Build the packages: `yarn release:build`.
 - [ ] Release the versions on npm: `yarn release:publish` (you need your 2FA device).
 - [ ] Push the newly created tag: `yarn release:tag`.
```

Comment:
The second one requires OTP.
Maybe it could disrupt the execution of the last command?
Also, all these three commands are very important to get right and any errors need to be clearly understood by the executor. I feel that running them all in one batch might disorient someone doing a release for the first or the second time.
Also 2: I believe the `yarn release:tag` usually errors out (did not check why) and we need to push the tag manually with the force flag. Having this error at the end might cause useless stress second guessing what has actually errored out.

IMHO: It's not a process you should rush doing and save every second optimising it's execution. 🙈

Reply:
`yarn release:tag` never work on my side as well, I'm always doing it manually.